demo/src/screens/componentScreens/SegmentedControlScreen.tsx

```diff
@@ -66,6 +72,14 @@ const SegmentedControlScreen = () => {
           segments={segments.seventh}
           segmentLabelStyle={styles.customTypography}
         />
+        <Text marginT-s4 center>
```

Comment:
Why no simply use the first SegmentedControl to switch btw the default and form presets on all/most of the examples?